src/common/types/feed.js

```diff
@@ -88,7 +96,32 @@ type AppActivityItems = {
     total_count: number,
 };
 
-type FeedItem = Comment | Task | BoxItemVersion | AppActivityItem;
+type Reply = {
+    createdAt: Date,
```

Comment:
Should these be snake_case? It looks like the generator I used for the Typescript types converted them to camelCase.

Reply:
I thought they should have, but followed the schema.

Reply:
The schema itself has them as snake_case. Probably check again?